### PR TITLE
Migrate playSound() in docs to melody.play()

### PR DESCRIPTION
**Pull request by ganicke**
Change usage of ``playSound`` and ``playSoundUntilDone`` to new melody api.

RE: #1016

### CODE REVIEW

**Comment by ganicke** [comment]
Build stalled:

```
No output has been received in the last 10m0s, this potentially indicates a stalled build or something wrong with the build itself.
```

**Comment by jwunderl** [comment]
I restarted the build on travis, looks like it's running fine for now so probably just a transient issue?

File: docs/courses/maker/projects/light-bot.md [hunk 1]
```diff
@@ -127,7 +127,7 @@ input.onSwitchMoved(SwitchDirection.Left, function () {
         servos.A2.run(-50)
         pause(1000)
         light.showAnimation(light.runningLightsAnimation, 500)
-        music.playSoundUntilDone(music.sounds(Sounds.Siren))
+        music.siren.play()
```
Comment (jwunderl) [inline docs/courses/maker/projects/light-bot.md]:
```suggestion
        music.siren.playUntilDone()
```

File: docs/learnsystem/pins-tutorial/digital-input/pin-events.md [hunk 2]
```diff
@@ -166,7 +166,7 @@ As an example, you could have a micro switch next to a doorway that breaks a clo
 ```blocks
 pins.A1.onEvent(PinEvent.Fall, function () {
     light.showAnimation(light.rainbowAnimation, 500)
-    music.playSoundUntilDone(music.sounds(Sounds.Wawawawaa))
+    music.wawawawaa.play()
```
Comment (jwunderl) [inline docs/learnsystem/pins-tutorial/digital-input/pin-events.md]:
```suggestion
    music.wawawawaa.playUntilDone()
```

Comment (ganicke) [reply docs/learnsystem/pins-tutorial/digital-input/pin-events.md]: Thanks for catching that.

**Pull request merged**